src/test/java/com/iota/iri/service/spentaddresses/impl/SpentAddressesServiceImplTest.java

```diff
@@ -0,0 +1,44 @@
+package com.iota.iri.service.spentaddresses.impl;
+
+import com.iota.iri.BundleValidator;
+import com.iota.iri.TangleMockUtils;
+import com.iota.iri.controllers.TransactionViewModel;
+import com.iota.iri.service.snapshot.SnapshotProvider;
+import com.iota.iri.service.spentaddresses.SpentAddressesProvider;
+import com.iota.iri.storage.Tangle;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'org.mockito.Mockito'](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4148367)

Reply:
@kwek20 , just deal with this please

Reply:
@GalRogozinski what do you mean here? i just github merged so if it changed not through this pr! I have since updated the style guidelines on my eclipse so we should be good next time

Reply:
I saw codacy comment, so I shouted...
This is a new file, and you are now in charge of this PR since Luca is not here, so can you please fix?

Reply:
Ahh never mind, it looks like it is a false positive